<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import</class>
 <widget class="QDialog" name="Import">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>297</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>297</width>
    <height>403</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>297</width>
    <height>403</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Demented - Import</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Catalog Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_catNum">
       <property name="text">
        <string>99999</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Band:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_band"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Album:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_album"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Base Price:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_basePrice"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_Location"/>
   </item>
   <item row="6" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QSpinBox" name="spinBox_XS"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>XS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBox_S"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBox_M"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>M</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QSpinBox" name="spinBox_L"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>L</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSpinBox" name="spinBox_XL"/>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>XL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSpinBox" name="spinBox_XXL"/>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>XXL</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Total Quantity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_TQ"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Notes:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QTextEdit" name="textEdit_Notes"/>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="4">
    <widget class="QPushButton" name="pushButton_import">
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QComboBox" name="comboBox_Type"/>
   </item>
  </layout>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>lineEdit_band</zorder>
  <zorder>lineEdit_album</zorder>
  <zorder>label_7</zorder>
  <zorder>textEdit_Notes</zorder>
  <zorder>comboBox_Type</zorder>
  <zorder>lineEdit_basePrice</zorder>
  <zorder>lineEdit_Location</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>pushButton_cancel</zorder>
  <zorder>pushButton_import</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
